comixed-frontend/src/app/library/components/convert-comics-settings/convert-comics-settings.component.ts

```diff
@@ -45,7 +45,8 @@ export class ConvertComicsSettingsComponent implements OnInit, OnDestroy {
     this.conversionForm = this.formBuilder.group({
       archiveType: ['', Validators.required],
       renamePages: ['', Validators.required],
-      deletePages: ['', Validators.required]
+      deletePages: ['', Validators.required],
+      deleteOriginalComic: ['', Validators.required]
```

Comment:
Given that it's a binary checkbox, does it affect the form to make this a required field?

Reply:
After testing, with binary checkbox, required field is not necessary.